ament_cmake_version/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Is there a need to require 3.10? If not maybe relaxing it to at least `3.7` would be good to have the option to backport it to Dashing.